README.md

```diff
@@ -45,17 +45,26 @@ All members that have been inducted for standard allowance as per the [Fellowshi
 All members must serve a minimum period of service of 12 months at their current rank before seeking promotion to a higher rank. 
 
 
+## Timelines
+
+Members can individually monitor the progress of their demotion period on the [Core Fellowship UI](https://collectives.subsquare.io/fellowship/core) provided by Subsquare. Members will soon be able to set up an ical feed to set reminders and get updates for their membership status.
```

Comment:
webcal://fellowship-calendar.kchr.de/?account=YOUR_ACCOUNT_ID we could already mention it.

Reply:
> webcal://fellowship-calendar.kchr.de/?account=YOUR_ACCOUNT_ID we could already mention it.

I have added the info for the ical feed. Please advise if anything needs to be changed.

Reply:
Looks good.